basketserver/src/test/java/de/upb/crypto/incentive/basketserver/BasketserverApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package de.upb.crypto.incentive.basketserver;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class BasketserverApplicationTests {
+
+    @Test
+    void contextLoads() {
+    }
+
+}
```

Comment:
Is this a class that is yet to be implemented?

Reply:
No, this test fails if the application context does not load and succeeds otherwise. Looks kind of empty though :)